usr/src/uts/common/fs/zfs/zrlock.c

```diff
@@ -21,6 +21,7 @@
 /*
  * Copyright (c) 2010, Oracle and/or its affiliates. All rights reserved.
  * Copyright (c) 2014, 2015 by Delphix. All rights reserved.
+ * Copyright 2016 The MathWorks, Inc. All rights reserved.
```

Comment:
Did you implement this independently or are you publishing a fix you found (e.g. Will's webrev from 3 years ago, or the other fixes folks seemed to have had in their trees forever)? If it is not your implementation, the copyright should have the proper attribution (e.g. Spectra Logic if you're pushing the code Will published).